css/style.css

```diff
@@ -0,0 +1,509 @@
+@import url('https://fonts.googleapis.com/css2?family=PT+Serif:wght@400;700&family=Roboto&display=swap');
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+body {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  box-sizing: border-box;
+}
+
+.wrap__site {
+  min-width: 375px;
+  max-width: 1920px; 
+  width: 100%;
+  height: 100vh;
+  margin: 0 auto;
+  display: flex;
+}
+
+/*header*/
+.header__burger {
+  display: none;
+}
+
+.header{
+  display: flex;
+  height: 35px;
+  max-width: 1550px;
+  justify-content: space-between;
+  margin: 41px auto 168px;
+}
+
+.header_logo-icon {
+  display: flex;
+}
+
+.logo-icon__item {
+  width: 28px;
+  height: 28px; 
+  border-radius: 50%;
+  opacity: .7;
+}
+
+.logo-icon__item-blue {
+  position: relative;
+  z-index: 1;
+  background-color: #0056EC;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#0056EC" to be "#0056ec" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/miniProject/pullRequest?prid=5593059)